packages/neo-one-website/docs/0-installation/2-environment-setup.md

```diff
@@ -35,23 +35,28 @@ This page describes how to setup NEO•ONE using `yarn` or `npm`.
 ```json
 {
   "sdk": {
-    "version": "3.1.401"
+    "version": "3.1.404"
   }
 }
 ```
 
-This tells your local C# .NET runtime to use version 3.1.401 in this repo, even if you have newer versions installed on your machine.
+This tells your local C# .NET runtime to use version 3.1.404 in this repo. Make sure you are using versions 3.1.4xx. You can find out the current installed version by typing `dotnet --version` on the console. This is required because NEO•ONE node uses the official C# NeoVM under the hood.
```

Comment:
I believe you actually want to run `dotnet --list-sdks` to get the list of installed versions. And run `dotnet --version` to see what version dotnet is currently using. It will use the latest version unless it sees the `global.json`. So `dotnet --version` should be run in the working repo to make sure dotnet is reading the `global.json`